### PR TITLE
Update Web App Icons to Natural Boxing

### PR DESCRIPTION
**Pull request by ctsims**
Since the web app icon tiles naturally resize, their [layout was very hard to make consistent](https://github.com/dimagi/commcare-hq/pull/27825#issuecomment-650232968), and it was difficult to figure out exactly the right margins for the layout.

This updates the web icons to be boxed naturally with the same margins and padding as the default flower icons, which means an uploader can upload an icon that fills a box and have it look natural without guessing at the margins.

Here's the result of naively laid out images which were resized by the layout engine. The only clear tradeoff remaining is if a user uploads an image that is < 150px in either dimension (the blurry first image which gets upsampled) which I'm fine forcing the burden downward to add manual margins
image

(old behavior for reference with these same icons)
image

### CODE REVIEW

File: corehq/apps/hqwebapp/static/cloudcare/less/formplayer-common/appicon.less [hunk 1]
```diff
@@ -22,6 +22,10 @@
       line-height: @icon-height;
       font-size: @icon-height;
     }
+    .appicon-custom {
+      top: 0.05*@icon-height + 10;
```
Comment (ctsims) [inline corehq/apps/hqwebapp/static/cloudcare/less/formplayer-common/appicon.less:26]: Note: This 10px buffer is there to replicate the fact that the CC flower right now has its own internal / natural padding on top of the layout.

Without this, there's a really awkward situation where users need to add a default padding to the top of their image but not the bottom, which is a very weird thing to force them to do

Comment (orangejenny) [reply corehq/apps/hqwebapp/static/cloudcare/less/formplayer-common/appicon.less:26]: I like that. The one production app I've seen using this has an icon that bumps up against the top of the tile and it doesn't look right.

File: corehq/apps/cloudcare/templates/formplayer/grid_view.html [hunk 2]
```diff
@@ -111,10 +111,13 @@ <h3>{% trans "Settings" %}</h3>
   <div class="container">
     <div class="spacer"></div>
     <div class="col-xs-6 col-sm-4 col-xs-offset-3 col-sm-offset-4">
-      <div class="appicon appicon-default js-start-app" <% if (imageUrl) { %>style="background-image: url('<%= imageUrl %>');"<% } %>>
+      <div class="appicon appicon-default js-start-app">
         <% if (!imageUrl) { %>
           <i class="fcc fcc-flower appicon-icon" aria-hidden="true"></i>
         <% } %>
+        <% if (imageUrl) { %>
```
Comment (orangejenny) [inline corehq/apps/cloudcare/templates/formplayer/grid_view.html:118]: Nitpick: not sure if you're intentionally using a second `if`, but `else` does work here, if you change the previous line to `<% } else { %>`

**Review by orangejenny — APPROVED** [review]

File: corehq/apps/hqwebapp/static/cloudcare/less/formplayer-common/appicon.less (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/hqwebapp/static/cloudcare/less/formplayer-common/appicon.less:26]: Having said that, can you move this rationale into a comment?

Comment (ctsims) [reply corehq/apps/hqwebapp/static/cloudcare/less/formplayer-common/appicon.less:28]: Good call. done

File: corehq/apps/cloudcare/templates/formplayer/grid_view.html (hunk 2 above)
Comment (ctsims) [reply corehq/apps/cloudcare/templates/formplayer/grid_view.html]: Ah, I was not. good call. Fixed

**Review by orangejenny — APPROVED** [review]

**Pull request merged**